commands/info.md

```diff
@@ -148,6 +148,7 @@ Here is the meaning of all fields in the **memory** section:
 *   `lazyfree_pending_objects`: The number of objects waiting to be freed (as a
      result of calling `UNLINK`, or `FLUSHDB` and `FLUSHALL` with the **ASYNC**
      option)
+*   `lazyfreed_objects`: The number of objects that have been lazy freed since redis was started.
```

Comment:
`since redis was started` is not technically correct (considering CONFIG RESETSTAT).
i see other stats fields avoid this by just being vague (see `evicted_keys`)

Reply:
right, so what about i change it to `The number of objects that have been lazy freed.`?